parlai/chat_service/README.md

```diff
@@ -0,0 +1,65 @@
+# Setting up ParlAI for a custom chat-service
+This is an instruction manual to be used as reference on how to configure ParlAI for an arbitrary chat service.
+
+## Message Format
+To maintain consistency we are trying to enforce a deterministic message format throughout this task. If a particular chat service doesn't adhere to this format, one must use ```restructure_message()``` to adapt the messages to this format before the messages being used. The message format is defined below:
```

Comment:
nit: "before the messages are used". Also, could you provide a quick example of when one might need to restructure a message?

Reply:
So I did mention in the previous sentence the use case within that line, maybe it wasn't clear. My intent was to say that suppose an API has different keys in their message dict they should transform it so that they use these specific keys when dealing with messages when implementing the bot in ParlAI. Not sure if that's what you meant by a quick example, could you clarify if that isn't the case? Or maybe provide any suggestions on how to rephrase that in a way that is more clear ?

Reply:
ahh i just meant something like "For example, with the FB Messenger chat service, a message might need to be restructured to fit fields X,Y,Z." It's not necessary if you can't think of one off the cuff